contracts/GenArt721CoreV3.sol

```diff
@@ -196,6 +212,20 @@ contract GenArt721CoreV3 is ERC721, Ownable, IGenArt721CoreContractV3 {
         return thisTokenId;
     }
 
+    /**
+     * @notice Internal function that returns whether msg.sender is allowed to
+     * call function with selector `_selector`, as determined by the Admin ACL
+     * contract.
+     * @param _selector Function selector to check.
+     * @dev assumes the Admin ACL contract is the owner of this contract, which
+     * is expected to always be true.
+     */
+    function _adminAllowed(bytes4 _selector) internal returns (bool) {
+        return
+            owner() != address(0) &&
```

Comment:
This check probably warrants a @dev comment above or an inline comment for improved readability.

Reply:
Good callout! updated!